Source/SwiftLintBuiltInRules/Rules/Idiomatic/ConvenienceTypeRule.swift

```diff
@@ -214,17 +214,17 @@ private extension AttributeListSyntax? {
 
 private extension AttributeListSyntax? {
     var hasUnavailableAttribute: Bool {
-        guard let attrs = self else {
+        guard let attributes = self else {
             return false
         }
 
-        return attrs.contains { elem in
-            guard let attr = elem.as(AttributeSyntax.self),
-                  let arguments = attr.argument?.as(AvailabilitySpecListSyntax.self) else {
+        return attributes.contains { elem in
+            guard let attributes = elem.as(AttributeSyntax.self),
```

Comment:
This one should remain singular as it refers to a single attribute. It's also better to avoid shadowing.
```suggestion
            guard let attribute = elem.as(AttributeSyntax.self),
```